prover/src/trace/trace_lde/default/mod.rs

```diff
@@ -118,7 +118,7 @@ where
     /// This function will panic if any of the following are true:
     /// - the number of rows in the provided `aux_trace` does not match the main trace.
     /// - this segment would exceed the number of segments specified by the trace layout.
-    fn add_aux_segment(
+    fn set_aux_segment(
```

Comment:
I think the comment on line 120 needs to be updated.

Reply:
done